### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by layertwo**
As discussed, this is the correct branch with the correct commits for Python3 and Bootstrap 3.4.

This replaces PR #25. There may be other additional changes I made along that way that I can cherry pick in.

### CODE REVIEW

**Comment by wwsean08** [comment]
A quick look at this looks fine, I'm gonna pull this and play around with it and do a fuller review this weekend since it's it's relatively large

**Comment by wwsean08** [comment]
Currently testing and I'm seeing the title of the URL getting surrounded by b'' so you specify a title of foo and get b'foo'

**Comment by layertwo** [comment]
Ok, that should be fixed. Updated the `escapeascii` function, and it seems the encoding isn't needed in Python3.

File: go.py [hunk 1]
```diff
@@ -662,7 +651,7 @@ def _removeLinkFromUrls(self, url):
 
     def deleteList(self, LL):
         for link in list(LL.links):
-            L.removeLink(link)
```
Comment (wwsean08) [inline go.py]: I've gotta dig deeper about this but I'm curious why this change was made.  Other than this things look fine, I'm currently testing this out with production traffic.

Comment (layertwo) [reply go.py]: `L` is an undefined variable here. `LL` is not. I made that change assuming it was a typo.

Comment (wwsean08) [reply go.py]: Ah good to know, I'm amazed it worked all these years, I've got a test server running and am sending production traffic at it and as long as everything looks good with that I'll probably merge this in early next week.

**Comment by wwsean08** [comment]
Thanks for the contribution, merged.
